### PR TITLE
LPD-31389 Add test

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:echo

**Comment by victorg1991** [comment]
ci:test:echo

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/Sidebar.js [hunk 1]
```diff
@@ -30,32 +29,23 @@ import isSmallResolution from '../utils/isSmallResolution';
...
- * @param {object} sidebarPanels
- */
-const getActivePanelData = ({panelId, panels, sidebarPanels}) => {
-	let sidebarPanelId = panelId;
-
-	let panel = panels.some((panel) => panel.includes(sidebarPanelId))
-		? sidebarPanels[sidebarPanelId]
-		: null;
+export const MAX_SIDEBAR_WIDTH = 500;
+export const MIN_SIZEBAR_WIDTH = 280;
```
Comment (veroglez) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/Sidebar.js:33]: Although I really like the name of this variable, I think it's a typo XD

Comment (veroglez) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/Sidebar.js:33]: Although I really like the name of this variable, I think it's a typo XD

**Comment by victorg1991** [comment]
ci:test:echo

**Comment by victorg1991** [comment]
should be done now waiting for tests

**Comment by victorg1991** [comment]
Seems unrelated

**Comment by victorg1991** [comment]
ci:test:echo

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:stop

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:no:me:abortes:la:build

**Comment by victorg1991** [comment]
ci:forward

**Pull request closed without merging**